dockerfiles/slurm/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:latest
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Based on:
+#  - https://bitbucket.org/deardooley/agave-docker/src/master/test-containers/schedulers/slurm/Dockerfile
+#  - https://ubuntuforums.org/showthread.php?t=2404746
+#  - https://stackoverflow.com/questions/46966876/installing-emulating-slurm-on-an-ubuntu-16-04-desktop-slurmd-fails-to-start
+
+RUN apt-get update && apt-get install -y \
+    lsof \
```

Comment:
The test battery uses this to look at running jobs I think. The container failed locally when I wrote the functional test and complained that it couldn't find `lsof`.

Reply:
FYI

Cylc is supposed to be able to run on POSIX systems provided GNU coreutils are installed.

The test battery, however, depends on a few more GNU utils, including `sed`, `awk` and apparently `lsof`.